docs/tasks/tools/install-kubectl.md

```diff
@@ -187,8 +187,16 @@ kubectl completion bash > $(brew --prefix)/etc/bash_completion.d/kubectl
 
 The Homebrew project is independent from Kubernetes, so the bash-completion packages are not guaranteed to work.
 
-### Using Oh-My-Zsh
-When using [Oh-My-Zsh](http://ohmyz.sh/), edit the ~/.zshrc file and update the `plugins=` line to include the kubectl plugin.
+### Using Zsh
+If you are using zsh edit the ~/.zshrc file and add the following code to enable kubectl autocompletion:
+
```

Comment:
Suggestions:

If you are using zsh, edit the `~/.zshrc file` ... 

edit the `~/.zshrc file` ...